### PR TITLE
Align Metric Family Descriptions

### PR DESCRIPTION
**Pull request by amnonh**
Metrics families (e.g., all metrics with the same name but with different labels) should have the same description.
The metric layer does not enforce that. Instead, it will use the first description provided.
It's a minor issue but the results are different than what you expect.

No need to backport.

### CODE REVIEW

File: service/storage_proxy.cc [hunk 1]
```diff
@@ -2801,15 +2801,15 @@ void storage_proxy_stats::stats::register_stats() {
                        {storage_proxy_stats::current_scheduling_group_label()}).set_skip_when_empty(),
 
         sm::make_total_operations("reads", replica_data_reads,
-                       sm::description("number of remote data read requests this Node received"),
+                       sm::description("number of remote reads this Node received. op_type labek could be data, mutation_data or digest"),
```
Comment (mykaul) [inline service/storage_proxy.cc:2804]:
```suggestion
                       sm::description("number of remote reads this Node received. op_type label could be data, mutation_data or digest"),
```

File: service/storage_proxy.cc [hunk 2]
```diff
@@ -2801,15 +2801,15 @@ void storage_proxy_stats::stats::register_stats() {
...
                        {storage_proxy_stats::current_scheduling_group_label(), storage_proxy_stats::op_type_label("data")}).set_skip_when_empty(),
 
         sm::make_total_operations("reads", replica_mutation_data_reads,
-                       sm::description("number of remote mutation data read requests this Node received"),
+                       sm::description("number of remote reads this Node received. op_type labek could be data, mutation_data or digest"),
```
Comment (mykaul) [inline service/storage_proxy.cc:2808]:
```suggestion
                       sm::description("number of remote reads this Node received. op_type label could be data, mutation_data or digest"),
```

File: service/storage_proxy.cc [hunk 3]
```diff
@@ -2801,15 +2801,15 @@ void storage_proxy_stats::stats::register_stats() {
...
                        {storage_proxy_stats::current_scheduling_group_label(), storage_proxy_stats::op_type_label("mutation_data")}).set_skip_when_empty(),
 
         sm::make_total_operations("reads", replica_digest_reads,
-                       sm::description("number of remote digest read requests this Node received"),
+                       sm::description("number of remote reads this Node received. op_type labek could be data, mutation_data or digest"),
```
Comment (mykaul) [inline service/storage_proxy.cc:2812]:
```suggestion
                       sm::description("number of remote reads this Node received. op_type label could be data, mutation_data or digest"),
```

**Comment by tchaikov** [comment]
> #### 🔴 CI State: FAILURE
> 
> ✅ - [Build](https://jenkins.scylladb.com/job/scylla-master/job/scylla-ci/10641/) ✅ - [Container Test](https://jenkins.scylladb.com/job/scylla-master/job/scylla-ci/10641/) ❌ - [dtest](https://jenkins.scylladb.com/job/scylla-master/job/gating-dtest-release/10851/) ✅ - [dtest with topology changes](https://jenkins.scylladb.com/job/scylla-master/job/gating-dtest-release-with-consistent-topology-changes/2830/) ✅ - [Unit Tests](https://jenkins.scylladb.com/job/scylla-master/job/scylla-ci/10641/)
> #### Failed Tests (1/31541):
> 
>     * [test_ignore_dead_nodes_for_whole_dc_replace_option](https://jenkins.scylladb.com//job/scylla-master/job/scylla-ci/10641/testReport/junit/repair_based_node_operations_test/TestRepairBasedNodeOperations/Tests___dtest___test_ignore_dead_nodes_for_whole_dc_replace_option) [🔍](https://github.com/scylladb/scylladb/issues?q=is:issue+is:open+test_ignore_dead_nodes_for_whole_dc_replace_option)
> 
> 
> #### Build Details:
> 
>     * Duration: 5 hr 38 min
> 
>     * Builder: spider4.cloudius-systems.com

#19907

**Comment by avikivity** [comment]
> "Metric family should have the description" 

Please follow our normal imperative mood for subjects. The patch is doing something to the code, describe that something. Not a philosophical statement about families.

**Comment by amnonh** [comment]
> 
> Please follow our normal imperative mood for subjects. The patch is doing something to the code, describe that something. Not a philosophical statement about families.

I agree. A Tolstoy reference would have been in place

**Review by nyh — APPROVED** [review]

**Pull request closed without merging**